apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/HHAClaimTransformerTest.java

```diff
@@ -60,13 +86,14 @@ public void transformSampleARecord() throws FHIRException, IOException {
    * @param claim the {@link HHAClaim} that the {@link ExplanationOfBenefit} was generated from
    * @param eob the {@link ExplanationOfBenefit} that was generated from the specified {@link
    *     HHAClaim}
-   * @param transformerContext the {@link TransformerContext} that was generated from the specified
-   *     {@link HHAClaim}
    * @throws FHIRException (indicates test failure)
    */
-  static void assertMatches(
-      HHAClaim claim, ExplanationOfBenefit eob, TransformerContext transformerContext)
-      throws FHIRException {
+  public static void assertMatches(HHAClaim claim, ExplanationOfBenefit eob) throws FHIRException {
+    // interesting conumdrum here....we should be using Mock(s) for unit tests, but this static
+    // method is invoked from ITs (integration tests) which means that our BeforeEach setup will
+    // not create the NPIOrgLookup so we need to explicitly create one here.
+    NPIOrgLookup localNpiLookup = new NPIOrgLookup();
```

Comment:
Is this cause createNpiOrgLookup gets called to essentially set up a singleton NpiOrgLookup via the fileInputStream? 

You should be able to mock the static method by doing something like:
```
try (MockedStatic<NPIOrgLookup> npiLookup = Mockito.mockStatic(NPIOrgLookup.class)) {
        npiLookup.when(NPIOrgLookup::createNpiOrgLookup).thenReturn(mockNpiLookup);
    }
```
(see https://www.baeldung.com/mockito-mock-static-methods )

Reply:
After discussion deferred mocking of npi and drug code to https://jira.cms.gov/browse/BFD-2766 since its currently not done in the V2 transformer tests either